pennylane/qaoa/cycle.py

```diff
@@ -14,8 +14,8 @@
 r"""
 Functionality for finding the maximum weighted cycle of directed graphs.
 """
-
```

Comment:
Reminder for changelog :smile:

Reply:
updated WIP entry here: 4f616048847885538c35a39ab38f216d8490e3d8